src/color.js

```diff
@@ -0,0 +1,48 @@
+
+//rgba() format use the following ranges: [0-255] for RGB and [0-1] for the alpha channel
+
+function marshall(color) {
+    return 'rgba(' + Math.round(color.r) + ',' + Math.round(color.g) + ',' + Math.round(color.b) + ',' + color.a + ')';
+}
+module.exports.marshall = marshall;
+
+function unmarshall(color, reference = null) {
+    function hexToInt(hex) {
+        return parseInt('0x' + hex);
+    }
+    var result = {};
+    if (color[0] === '#') {
+        result.r = hexToInt(color.substr(1, 2));
+        result.g = hexToInt(color.substr(3, 2));
+        result.b = hexToInt(color.substr(5, 2));
+        result.a = 1;
+    } else if (color.startsWith('rgb(')) {
```

Comment:
Will we need a polyfill to use `startsWith`?

Reply:
It's a good question, I could look at `startsWith` docs, but I've worked without limiting myself to a particular ES version. Is there a way to automatically detect which ES version works? Do we want to configure babel?